netebpfext/net_ebpf_ext_sock_addr.c

```diff
@@ -692,7 +692,7 @@ _net_ebpf_ext_connection_context_initialize_key(
 }
 
 static NTSTATUS
-_net_ebpf_ext_get_connection_context(
+_net_ebpf_ext_get_and_delete_connection_context(
```

Comment:
you are removing the context from the list, not deleting it, at least not in this function. So please rename.

Reply:
Renamed.